spec/features/reviews/default_picture_spec.rb

```diff
@@ -0,0 +1,16 @@
+# require 'rails_helper'
+#
+# RSpec.describe "default review image" do
+#     it "default image is used if user does not sumbit image" do
+#
+#   visit "/shelters/#{@shelter_1.id}/reviews"
+#
+#   fill_in 'Title', with: "Test"
+#   fill_in 'Rating', with: 5
+#   fill_in 'Content', with: "Test"
+#
+#   click_on 'Submit'
+#
+#   expect(page).to have_xpath("//img[contains(@src,'https://www.eaglenewsonline.com/wp-content/uploads/2018/12/paws-tongue.jpg')]")
+# end
```

Comment:
Let us remember to come back to this one

Reply:
I have notes on which tests need to be working. Added and cried